mbq/atomiq/__init__.pyi

```diff
@@ -0,0 +1,10 @@
+from typing import Callable
+
+from mbq import metrics
+
+
+sns_publish: Callable[..., None]
+sqs_producer: Callable[..., None]
+celery_publish: Callable[..., None]
+
+_collector: metrics.Collector
```

Comment:
I'm 👍 shipping this for now but let's add a note to revisit the approach. We can roll out a change to how we do this without having to updating all the apps.

Reply:
I'm with you on that: https://app.clubhouse.io/managedbyq/story/30741/revisit-the-way-we-initialize-our-library-django-apps